test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -339,7 +339,7 @@ public void BinaryClassifierLogisticRegressionBinNormTest()
         /// <summary>
         ///A test for binary classifiers
         ///</summary>
-        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 output differs from Baseline
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCoreAnd64BitProcess))] // x86 output differs from Baseline and flaky on netcore 3.0
         [TestCategory("Binary")]
         public void BinaryClassifierLogisticRegressionGaussianNormTest()
```

Comment:
Are you adding these 2 tests to the list of tests to investigate?

Reply:
i will add the it to the list